example-dmn-runtime-service/build.properties

```diff
@@ -1,3 +1,3 @@
-DOCKER_IMAGE_NAME=example-dmn-runtime-service
+DOCKER_IMAGE_NAME=${DOCKER_IMAGE_NAME:-$(basename $PWD)}
```

Comment:
it is not totally clear to me the whole `${DOCKER_IMAGE_NAME:-$(basename $PWD)}`
Is it necessary? Can i be simplified?

Reply:
it is actually better as it takes the name from the directory and it is the same everywhere

Reply:
It is better if the customer would use this exactly like it is, but i doubt the use the same trick of the folder name in production.

So on my opinion i am not sure that it is better, but definitely it is not clearer as there is more scripting and the reader needs to get the logic you wanted to apply

Reply:
I would prefer to keep it as is so you don't forget to update it